src/applications/vaos/services/appointment/transformers.js

```diff
@@ -114,9 +114,11 @@ export function transformVAOSAppointment(appt, useFeSourceOfTruth) {
   const isVideo = appt.kind === 'telehealth' && !!appt.telehealth?.vvsKind;
   const isAtlas = !!appt.telehealth?.atlas;
   const isPast = isPastAppointment(appt);
-  const isRequest =
-    appointmentType === APPOINTMENT_TYPES.request ||
-    appointmentType === APPOINTMENT_TYPES.ccRequest;
+  // todo: need to cleanup when using api response to determine a cc request
+  const isRequest = useFeSourceOfTruth
+    ? appt.pending || appointmentType === APPOINTMENT_TYPES.ccRequest
```

Comment:
Hmm I don't think this looks right. The backend sets appt.pending for both va request and cc request. Here's [the code](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/vaos/app/services/vaos/v2/appointments_service.rb#L650-L654) that sets it. Given that that's the case, pending should be mocked to "true" for all requests, regardless of VA request or CC request.

Reply:
Okay, will revert to previous.